app/LoadedApp/LoadedApp.tsx

```diff
@@ -2125,20 +2107,26 @@ export class LoadedAppClass extends Component<LoadedAppClassProps, LoadedAppClas
           (valueTransfersTotal !== null && valueTransfersTotal > 0) ||
           (!readOnly && !!totalBalance && totalBalance.spendableOrchard + totalBalance.spendablePrivate > 0) ? (
             <Tab.Navigator
+              detachInactiveScreens={true}
               initialRouteName={translate('loadedapp.history-menu') as string}
-              screenOptions={({ route, navigation }) => ({
-                tabBarIcon: ({ focused }) => fnTabBarIcon(route, focused, navigation),
+              screenOptions={({ route }) => ({
+                tabBarIcon: ({ focused }) => fnTabBarIcon(route, focused),
+                tabBarIconStyle: { alignItems: 'center', justifyContent: 'center', alignSelf: 'center' },
+                tabBarItemStyle: {
+                  justifyContent: 'center',
+                  alignSelf: 'center',
+                },
                 tabBarLabelPosition: 'below-icon',
-                tabBarActiveTintColor: 'transparent',
+                tabBarActiveTintColor: colors.background,
                 tabBarActiveBackgroundColor: colors.primaryDisabled,
                 tabBarInactiveTintColor: colors.money,
-                tabBarShowLabel: false,
                 tabBarStyle: {
                   borderRadius: 0,
                   borderTopColor: colors.primaryDisabled,
                   borderTopWidth: 1,
                 },
                 headerShown: false,
+                tabBarButton: props => <PlatformPressable {...props} android_ripple={{ color: colors.primary }} />,
```

Comment:
Advice from lint:
Do not define components during render. React will see a new component type on every render and destroy the entire subtree’s DOM nodes and state (https://reactjs.org/docs/reconciliation.html#elements-of-different-types). Instead, move this component definition out of the parent component “LoadedAppClass” and pass data as props. If you want to allow component creation in props, set allowAsProps option to true.

I think can be a good idea if you wrap this outside of the render...